osu.Game.Rulesets.Catch/UI/Catcher.cs

```diff
@@ -35,7 +37,30 @@ public class Catcher : Container, IKeyBindingHandler<CatchAction>
 
         public Container ExplodingFruitTarget;
 
-        public Container AdditiveTarget;
+        private Container additiveTarget;
+        private Container<CatcherTrailSprite> dashTrails;
+        private Container<CatcherTrailSprite> hyperDashTrails;
+        private Container<CatcherTrailSprite> endGlowSprites;
```

Comment:
Not sure I like these being loose and out in the open in the catcher like this. Might make sense to encapsulate in some sort of private subcontainer class to reduce noise in the catcher class.

Reply:
just to be clear, do you mean to put these 3 trails containers inside a private container with public accessibility to all containers inside and access them on all usages?

Reply:
I think this is fine.